eqcorrscan/core/match_filter/family.py

```diff
@@ -326,8 +326,7 @@ def _uniq(self):
         2
         """
         _detections = []
-        [_detections.append(d) for d in self.detections
-         if not _detections.count(d)]
+        _detections = list(set(self.detections))
```

Comment:
Making a `set` of detections relies on them being hashable, which they are not (or should not be - they can and do change - I did accidentally make them hashable at one point). Therefore I don't think this method should work. If it isn't returning the following error:
```python
TypeError: unhashable type: 'Detection'
```
then it should!

Reply:
Thanks for pointing this out.. indeed I didn't get the error in any of my tests which is surprising considering that these errors came up in all tests on github. I'll revert this change here